java/src/main/java/ai/onnxruntime/ONNXUtil.java

```diff
@@ -0,0 +1,674 @@
+/*
+ * Copyright © 2019, Oracle and/or its affiliates. All rights reserved.
+ * Licensed under the MIT License.
+ */
+package ai.onnxruntime;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Util code for interacting with shape arrays.
+ */
+public class ONNXUtil {
+    /**
+     * Creates a new primitive boolean array of up to 8 dimensions, using the supplied shape.
+     * <p>
+     * Does not check the shape to see if all it's elements are positive.
+     *
+     * @param shape The shape of array to create.
+     * @return A boolean array.
+     */
+    public static Object newBooleanArray(long[] shape) {
+        switch (shape.length) {
```

Comment:
I know that you need to handle long[] in the native code to call more easily the onnxruntime, but you could remove a lot of code if you had a int[] here  
```java
public static Object newBooleanArray(int[] shape) {
  return Array.newInstance(boolean.class, shape);
}
```

Reply:
Agreed,

I’ve tried to handle it in the native code but had some issues with the int* cast to *long but we have alignement issue. 

easier to do it in the java api but maybe less performant

Reply:
Done, thanks.